tyr-core/src/main/resources/format-example.ci.yaml

```diff
@@ -0,0 +1,33 @@
+repository: "jboss/test-repo"
```

Comment:
use format-example-ci.yaml, I'm not sure but . is an end delimiter so it might be wrongly parsed somewhere